RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
```

Comment:
We don't need a release file for changes to the docs, they're published directly after a PR is merged and don't require a full package release